src/core/Str.pm

```diff
@@ -178,6 +188,8 @@ my class Str does Stringy { # declared in BOOTSTRAP
         )
     }
 
+    proto method indices(|) {*}
+    multi method indices(Str:D: Cool:D $needle, *%pars) {self.indices: $needle.Str, |%pars}
     multi method indices(Str:D: Str:D $needle, :$overlap) {
```

Comment:
The Cool had `multi method indices(Cool:D: Str(Cool) $needle, Int(Cool) $start, :$overlap) {` candidate, but there appears to be no candidate that takes `$start` argument in Str. 

Should it be added or was the Cool one a leftover that should've been removed?